regolith/helpers/u_finishprumhelper.py

```diff
@@ -65,15 +71,15 @@ def db_updater(self):
             return
         found_projectum.update({'status':'finished'})
         if rc.pub_sum:
-            found_projectum['deliverable'].update({'public_summary': rc.pub_sum})
-        elif found_projectum['deliverable'].get('public_summary') is not None:
-            print("Error: please enter the public summary by specifying pub_sum argument.")
-            return
+            found_projectum['public_summary'] = rc.pub_sum
+        elif found_projectum.get('public_summary') is not None:
```

Comment:
I think this logic is not quite right, please see my code.  The outer loop should check if the summaries exist in the db or not and it should not overwrite them if they do.  Then it should check if rc.pub_sum exists and use that, finally it should raise a runtime error if it neither already exists in the db nor has been given to the cli.

We should also catch the case where it finds something in the db and is given a value on the cli with a f"WARNING: public summary {found_projectum.get('public_summary')} exists in the db and will not be overwritten with {rc.pub_sum} given on the command line.  If you want to force this change please rerun adding --force" or sthg like that.